spec/chips_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'chips'
+
+describe Chips do
```

Comment:
chips is a child of dish and all the tests seem to be the same for both classes, but at the same time chips is a different class - I guess I'm wondering if you need these tests or what behaviour you can assume by it being a child of dish